### DISCUSSION TITLE
How to add a game to Steam Flatpak?

### DISCUSSION THREAD

**Discussion by CleoMenezesJr** [Heroic-Games-Launcher/HeroicGamesLauncher#4009] [Q&A] [answered]
```
Adding Grand Theft Auto V to Steam failed with:
~/.steam/steam/userdata does not exist. Can't add/remove game to/from Steam!

```

When trying to add to Steam, I receive the error message above, since Heroic apparently searches for the path to Steam installed on the system by default. Is there any way to make it exported to `~/.var/app/com.valvesoftware.Steam/data/Steam/userdata`?

**Comment by ironman820** [comment, marked as answer]
In the side menu, click on Settings -> General. About half way down that page is a setting labeled "Default Steam Path" in that box, enter `~/.var/app/com.valvesoftware.Steam/data/Steam` This will tell heroic where the steam folder is and update your steam library properly.

**Reply by CleoMenezesJr** [reply]
> `~/.var/app/com.valvesoftware.Steam/data/Steam`

I had to add the absolute path for it to work: `/var/home/cleomenezesjr/.var/app/com.valvesoftware.Steam/data/Steam`.

After that, everything worked fine, thanks,
